mtest/importmidi/updateReference

```diff
@@ -3,7 +3,7 @@
 if [ "`uname`" = 'Darwin' ]; then
    MSCORE=../../build.xcode/mtest/importmidi/Debug
 else
-   MSCORE=../../build.debug/mtest/importmidi
+   MSCORE=/home/presario/build-MuseScore-Desktop_Qt_5_9_9_GCC_64bit-Debug/mtest/importmidi
```

Comment:
that change is probably wrong

Reply:
IIRC I complained about that in another PR already ;-)